src/Basho/Riak/Bucket.php

```diff
@@ -208,13 +202,13 @@ public function newBinary($key = null, $data, $content_type = 'application/json'
      *
      * @param  string $key - Name of the key.
      * @param  int $r   - R-Value of the request (defaults to bucket's R)
+     *
      * @return Object
      */
     public function get($key, $r = null)
```

Comment:
Well, it still allows you to pass it through to the reload method to override the bucket - client values for that call. If I remove it, you wouldn't be able to do that from get().